doc/changes/changes_0.2.1.md

```diff
@@ -1,15 +1,42 @@
-# ai-lab 0.2.1, released 2024-??-??
+# AI-Lab 0.2.1, released 2024-??-??
 
-Code name: Fix cloud storage notebook
+Code name: Fix Cloud Storage Notebook
 
 ## Summary
 
+This release fixes the Cloud Storage notebook and also fixes vulnerabilities by updating dependencies in file `poetry.lock` and GitHub workflows.
+
+Impact and delimitation
+* Updating the dependencies required to upgrade the build environment from Python `3.8` to `3.10`.
+* Also AWS codebuild image need to be upgraded from `aws/codebuild/standard:5.0` to `6.0`.
+* The Jupyterlab notebooks and their libraries remain on Python `3.8` for now.
+
 ## AI-Lab-Release
 
 Version: 0.2.1
 
 ## Features
 
```

Comment:
```suggestion

- n/a

```

Reply:
I plan to remove all sections without any resp. tickets.